ideascube/library/migrations/0010_section_type.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-10-28 08:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+int_to_name = {1: 'digital',
+               2: 'children-cartoons',
+               3: 'children-novels',
+               10: 'children-poetry',
+               11: 'children-theatre',
+               4: 'children-documentary',
+               5: 'children-comics',
+               6: 'adults-novels',
+               7: 'adults-documentary',
+               8: 'adults-comics',
+               12: 'adults-poetry',
+               13: 'adults-theatre',
+               9: 'game',
+               99: 'OTHER'}
```

Comment:
This is the same order that the one in the model.
We may reorder the one in the model also.
However, field's choice in the model will evolve (as new sections are added) and not this one.
I don't have special idea on this point.